scripts/cm-rgb-monitor

```diff
@@ -114,19 +154,29 @@ def monitor(
             
         if showCPUFreq:
             freqs = psutil.cpu_freq(percpu=True)
-            max_freq = max(i.current for i in freqs)
-            smoothed_freq = fSmoothing * smoothed_freq + (1 - fSmoothing) * max_freq
-            interp_f = max(0, min(1, (smoothed_freq-2200)/(3700-2200)))            
+            maxFreq = max(i.current for i in freqs)
+            smoothedCPUFrequency = smoothing * smoothedCPUFrequency + (1 - smoothing) * maxFreq
+            interp_f = max(0, min(1, (smoothedCPUFrequency-2200)/(3700-2200)))            
```

Comment:
``psutil.cpu_freq`` returns min and max cpu frequency; perhaps 2200/3700 values should be replaced?
Wraith was available on few cpus with different frequency range.

Reply:
That's a good idea

Reply:
Potential fix:
`interp_f = max(0, min(1, (smoothedCPUFrequency-psutil.cpu_freq().min)/(psutil.cpu_freq().max-psutil.cpu_freq().min)))`

Though, that would change behaviour a bit:
If frequency is capped, e.g. for energy saving reasons (running BOINC on my machine, max 3200MHz), psutil.cpu_freq().max returns the cap value, not the default maximum value (3700MHz for my 2700X).

So it also breaks completely, if frequency is set to a fixed value --> 
`(psutil.cpu_freq().max-psutil.cpu_freq().min))` = 0 --> division by zero
But this is easily solved using try/except.

I think "fixing" this in this way isn't the ideal solution as frequency/color does not work any more on fixed frequencies and works _differently_ with manually changed frequency policies - though this might be an edge case.

What's your opinion?

```
try;
    interp_f = max(0, min(1, (smoothedCPUFrequency-psutil.cpu_freq().min)/(psutil.cpu_freq().max-psutil.cpu_freq().min)))
except ZeroDivisionError:
    interp_f = max(0, min(1, (smoothedCPUFrequency-2200)/(3700-2200)))
```

Reply:
a) Perhaps those could be optionally passed as arguments in situation like this?
b) This one is interesting...
I think I would add an if statement for max-min == 0; if its true then just set interp_f to either 0 or 1. 0 makes sense in my opinion as 1 represents more of an extreme situation.